plugins/main/public/components/common/search-bar/use-search-bar.ts

```diff
@@ -123,10 +140,16 @@ const useSearchBarConfiguration = (
     ): void => {
       const { dateRange, query } = payload;
       // its necessary execute setter to apply query filters
+      // when the hook receives the dateRange use the setter instead the global setTimeFilter
+      props?.setTimeFilter
+        ? props?.setTimeFilter(dateRange)
+        : setGlobalTimeFilter(dateRange);
+      props?.setQuery ? props?.setQuery(query) : setQuery(query);
+      props?.onQuerySubmitted && props?.onQuerySubmitted(payload);
       setTimeFilter(dateRange);
```

Comment:
**nitpick**: it seems the `setTimeFilter` method of `useTimefilter` was stored as another variable here: https://github.com/wazuh/wazuh-dashboard-plugins/pull/6482/files#diff-525ac530f57cf8ecd190735289664dd9283818a88314c75163c40a38c42520a0R47

Reply:
This is to use the received setters for query and time filters instead the global time filter and query service